Created landscape mode for CompletedScene Activity.

yatna opened the pull request:
Landscape mode has been created for CompletedSceneActivity.java  to fit all screen sizes.

![screenshot_2016-01-28-03-17-36](https://cloud.githubusercontent.com/assets/8321130/12830993/1c59f790-cbb8-11e5-986f-5871f1192a9b.png)

alma-castant commented on PowerUp/app/src/main/res/values/dimens.xml:
```diff
@@ -51,7 +51,7 @@
     <dimen name="powerup_width">304dp</dimen>
     <dimen name="power_bar_view_height">75dp</dimen>
-    <dimen name="game_text_size">20dp</dimen>
+    <dimen name="game_text_size">17dp</dimen>
```
Why is the text size reduced? This is causing an issue in the phone portrait mode of this screen (which should be refactored). We should not be introducing this here

alma-castant commented:
I left a comment. It looks good in the phone and tablet on portrait mode, but it makes phone portrait mode worse which should not be the case

yatna commented:
1.) Create new variables for text size in final screens.
2.) Restored game_text_size from 17 dp to 20 dp.

alma-castant commented on PowerUp/app/src/main/res/layout-land/completed_scene.xml:
```diff
@@ -0,0 +1,37 @@
+       android:gravity="center_horizontal"
+       android:layout_marginTop="@dimen/completed_game_textview">
+       <Button
+           android:id="@+id/ContinueButtonMap"
```
Be consistent with notation and name this `continue_button_map`

alma-castant commented on PowerUp/app/src/main/res/layout-land/completed_scene.xml:
```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
Instead of nesting two LinearLayouts, it would be better to use a RelativeLayout

alma-castant commented on PowerUp/app/src/main/res/values-land/dimens.xml:
```diff
@@ -37,5 +37,6 @@
     <dimen name="avatar_room_width">80dp</dimen>
     <dimen name="avatar_room_height">80dp</dimen>
     <dimen name="avatar_room_continue_button_land">30dp</dimen>
+	<dimen name= "completed_game_textview">50dp</dimen>
```
Use the same level of identation for all elements

alma-castant commented on PowerUp/app/src/main/res/values-sw600dp/dimens.xml:
```diff
@@ -38,5 +38,9 @@
     <dimen name="avatar_room_width">120dp</dimen>
     <dimen name="avatar_room_height">120dp</dimen>
     <dimen name="avatar_room_continue_button_land">80dp</dimen>
+	<dimen name="completed_game_textview">80dp</dimen>
```
Use the same level of identation for all elements

alma-castant commented on PowerUp/app/src/main/res/values-sw720dp/dimens.xml:
```diff
@@ -39,5 +39,9 @@
     <dimen name="avatar_room_width">150dp</dimen>
     <dimen name="avatar_room_height">150dp</dimen>
     <dimen name="avatar_room_continue_button_land">100dp</dimen>
+	<dimen name="completed_game_textview">140dp</dimen>
```
Same

alma-castant commented on PowerUp/app/src/main/res/values/dimens.xml:
```diff
@@ -52,6 +52,7 @@
     <dimen name="powerup_width">304dp</dimen>
     <dimen name="power_bar_view_height">75dp</dimen>
     <dimen name="game_text_size">20dp</dimen>
+	<dimen name="game_text_size_final">20dp</dimen>
```
Same

alma-castant commented on PowerUp/app/src/main/res/values/dimens.xml:
```diff
@@ -76,5 +77,6 @@
     <item name="map_alpha" type="dimen" format="float">0.00</item>
     <dimen name="avatar_room_margin">50dp</dimen>
     <dimen name="avatar_room_continue_button">50dp</dimen>
+	<dimen name= "completed_game_textview">50dp</dimen>
```
Same

yatna commented:
1. Corrected the indentation .
2. Corrected the naming (Corresponding naming in portrait mode and .java file is changed in another PR)
3. Creating a relative layout instead of a nested linear layout was introducing a lot of dimension constants  which had to adjusted according to different screen sizes. The portrait mode is currently a relative layout and does not work well with different screen sizes. Also the linear layout seems simple and easy to understand. Hence, I think it will be better to keep it as a linear layout .
